addons/block_code/types/types.gd

```diff
@@ -30,6 +30,16 @@ const STRING_TO_VARIANT_TYPE: Dictionary = {
 	"NIL": TYPE_NIL,
 }
 
+const VARIANT_TYPE_TO_GDSCRIPT_STRING: Dictionary = {
+	TYPE_STRING: "String",
+	TYPE_INT: "int",
+	TYPE_FLOAT: "float",
+	TYPE_BOOL: "bool",
+	TYPE_VECTOR2: "Vector2",
+	TYPE_COLOR: "Color",
+	TYPE_NODE_PATH: "NodePath",
```

Comment:
YES!!

Reply:
Isn't this just duplicating [type_string](https://docs.godotengine.org/en/stable/classes/class_%40globalscope.html#class-globalscope-method-type-string)? The following test passed for me:

```
extends GutTest

const VARIANT_TYPE_TO_GDSCRIPT_STRING: Dictionary = {
	TYPE_STRING: "String",
	TYPE_INT: "int",
	TYPE_FLOAT: "float",
	TYPE_BOOL: "bool",
	TYPE_VECTOR2: "Vector2",
	TYPE_COLOR: "Color",
	TYPE_NODE_PATH: "NodePath",
}

func test_type_string():
	for type in VARIANT_TYPE_TO_GDSCRIPT_STRING:
		var expected: String = VARIANT_TYPE_TO_GDSCRIPT_STRING[type]
		var type_str: String = type_string(type)
		assert_eq(type_str, expected)
```

Reply:
Correct! I don't know how I didn't think of that.